Sources/AWSLambdaEvents/APIGateway+V2.swift

```diff
@@ -56,6 +56,23 @@ public struct APIGatewayV2Request: Codable {
             public let lambda: LambdaAuthorizerContext?
         }
 
+        public struct Authentication: Codable {
+            public struct ClientCert: Codable {
+                public struct Validity: Codable {
+                    public let notBefore: String
+                    public let notAfter: String
+                }
+
+                public let clientCertPem: String
+                public let subjectDN: String
+                public let issuerDN: String
+                public let serialNumber: String
+                public let validity: Validity
+            }
+
+            public let clientCert: ClientCert?
+        }
+
```

Comment:
looks good, but can you add a small unit test as well?

Reply:
done